samples/android/app/build.gradle

```diff
@@ -0,0 +1,74 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-android-extensions'
+    id 'kotlin-kapt'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "org.hyperledger.indy.sdk.example"
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+
+    implementation files('libs/libindy-android-debug.aar')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.material:material:1.2.1'
+    implementation "androidx.constraintlayout:constraintlayout:2.0.2"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9"
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0'
+    implementation 'com.google.android.material:material:1.2.1'
+
+    // java
+    implementation "org.slf4j:slf4j-api:1.7.30"
+    implementation "net.java.dev.jna:jna:5.6.0@aar"
+    implementation "org.apache.commons:commons-lang3:3.11"
+    implementation("net.sourceforge.streamsupport:android-retrofuture:1.7.2") {
+        because("We want to support api level 21.")
+    }
+    implementation "commons-io:commons-io:2.7"
+
+    // logger
+    implementation('com.github.bright:slf4android:0.1.6'){
+        transitive = true
+    }
+
+    // view binding
+    android {
+        buildFeatures {
+            viewBinding = true
+        }
+    }
```

Comment:
`android` block is already defined in line 8. So add:
```
        buildFeatures {
            viewBinding = true
        }
```
In existing android block